keyboards/augio/keymaps/default/keymap.c

```diff
@@ -0,0 +1,96 @@
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______,     _______, 	   _______, _______, _______, _______, _______
+	),
+
+	[_LOWER] = LAYOUT(
+		_______, _______, _______, _______, RESET,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______,     _______, 	   _______, _______, _______, _______, _______
+	),
+
+	/* Adjust
+	 * ,----------------------------------------------------------------------------------------------------------------.
+	 * |      |      |      |      ||      |      |      |      |      |      |      |      |      |      |      |      |
+	 * |------+------+------+------++------+------+------+------+------+-------------+------+------+------+------+------|
+	 * |      |      |      |      ||      |      |      |      |      |      |      |      |      |      |      |      |
+	 * |------+------+------+------++------+------+------+------+------+-------------+------+------+------+------+------|
+	 * |      |      |      |      ||      |      |      |      |      |      |      |      |      |      |      |      |
+	 * |------+------+------+------++------+------+------+------+------+------+------+------+------+------+------+------|
+	 * |      |      |      |      ||      |      |      |      |      |             |      |      |      |      |      |
+	 * `----------------------------------------------------------------------------------------------------------------'
+	 */
+	[_ADJUST] = LAYOUT(
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______,     _______, 	   _______, _______, _______, _______, _______
+	)
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
+	//keyevent_t event = record->event;
+
+	switch (id) {
+
+	}
+	return MACRO_NONE;
+}
+
+void matrix_init_user(void) {
+}
+
+void matrix_scan_user(void) {
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+	return true;
+}
+
+void led_set_user(uint8_t usb_led) {
+
+}
```

Comment:
None of these are used or have been deprecated, and should be removed. 

```suggestion
```

Reply:
They are still here.